daiquiri/core/static/core/js/table.js

```diff
@@ -39,7 +39,11 @@ angular.module('core')
                 return row[column_index];
             },
             datalink_url: function(row, column_index) {
-                return service.datalink_url + row[column_index];
+                if (service.params.job) {
```

Comment:
this does not happen in my local instance

Reply:
So for some reasone this is False and the url is `service.datalink_url + row[column_index];` instead of `service.datalink_url + row[column_index] + '?job=' + service.params.job;`

Reply:
do I need to do an extra collect static maybe?

Reply:
depends on you setup, if you use the production setup, yes, if not, you don't need `collectstatic` at all ... so probably yes.